src/ui/VavatoTheme.js

```diff
@@ -1,6 +1,6 @@
 const VavatoTheme = {
-  primary: '#a29060',
-  secondary: '#6192a1',
+  primary: '#6192A1',
+  secondary: '#A29060',
```

Comment:
The switching of these 2 will mean all blue/gold colors inverted in apps using the vavato-ui right?

Reply:
Yes, as mentioned here: https://www.figma.com/file/8Ov6NBiWMkByan9xg275bB/Vavato-Internal-team-library?node-id=0%3A1 (page 2)